specviz/widgets/workspace.py

```diff
@@ -122,6 +122,9 @@ def __init__(self, *args, **kwargs):
         self._plugins = {}
         self._plugin_bars = {}
 
+        # Current working directory. Updated by file open dialogs
+        self.cwd = os.getcwd()
```

Comment:
Just a small quibble, but would you mind making this a private variable? E.g. `self._cwd`?

Reply:
Ok. See latest commit